src/common/database.c

```diff
@@ -1854,6 +1854,15 @@ static int _upgrade_data_schema_step(dt_database_t *db, int version)
 
     new_version = 6;
   }
+  else if(version == 6)
+  {
+    TRY_EXEC("CREATE TABLE data.locations "
+             "(tagid INTEGER PRIMARY KEY, type INTEGER, longitude REAL, latitude REAL, "
```

Comment:
`PRIMARY KEY` should be removed from here.

Reply:
I would agree on this.
Foreign key maintains the relationship with the master table.
Primary key guaranties the unicity of the key.

![image](https://user-images.githubusercontent.com/23012047/97861088-f941f200-1ce1-11eb-84d8-55b60d31a8a4.png)

Reply:
Sorry for the tipo. The primary key should stay.

Reply:
Ok, you're right !